Panic on sFlow from OpenVSwitch

Trying to enable sFlow from OVS with config:
`ovs-vsctl   --   --id=@s create sFlow agent=enp0s8  target=\"127.0.0.1:6343\" header=128 sampling=64 polling=10  -- set Bridge testbr sflow=@s`

This is the output from Cistern:
```

root@ubuntu:~# ./cistern
2017/01/04 17:27:04 main.go:54: Cistern version 0.0.4 starting
2017/01/04 17:27:04 clock.go:13:   Starting internal clock
2017/01/04 17:27:04 main.go:57:   Attempting to load configuration file at /opt/cistern/config.json
2017/01/04 17:27:04 main.go:60: ✗ Could not load configuration: `open /opt/cistern/config.json: no such file or directory`
2017/01/04 17:27:04 main.go:71: ✓ Successfully loaded configuration
2017/01/04 17:27:04 service.go:54: listening for sFlow datagrams on :6343
2017/01/04 17:27:04 service.go:59: listening for AppFlow datagrams on :6344
2017/01/04 17:27:04 main.go:90: ✓ Successfully started network service
2017/01/04 17:27:09 service.go:73: 10.0.3.15 is unknown. Registering new source.
2017/01/04 17:27:09 service.go:82: Source{10.0.3.15} needs class "sflow".
2017/01/04 17:27:09 source.go:34: registering class sflow for Source{10.0.3.15}
2017/01/04 17:27:09 source.go:66:   Source{10.0.3.15} does not have class "switch-counters" registered
2017/01/04 17:27:09 source.go:34: registering class switch-counters for Source{10.0.3.15}
2017/01/04 17:27:09 source.go:66:   Source{10.0.3.15} does not have class "metrics" registered
2017/01/04 17:27:09 source.go:34: registering class metrics for Source{10.0.3.15}
2017/01/04 17:27:09 engine.go:51: [Series engine] Writing 14 observations
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x4a98de]

goroutine 1 [running]:
panic(0x7b6180, 0xc82000a0f0)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
github.com/Cistern/cistern/state/series.(*Engine).writeObservations(0x0, 0xc8200da000, 0xe, 0x10)
	/Users/alexanderturner/Code/gocode/src/github.com/Cistern/cistern/state/series/engine.go:52 +0xde
github.com/Cistern/cistern/state/series.(*Engine).Process(0x0, 0xc820016a80)
	/Users/alexanderturner/Code/gocode/src/github.com/Cistern/cistern/state/series/engine.go:47 +0xae
main.main()
	/Users/alexanderturner/Code/gocode/src/github.com/Cistern/cistern/main.go:96 +0x1094
```

--- a/main.go
+++ b/main.go
@@ -57,18 +57,18 @@ func main() {
 	log.Printf("  Attempting to load configuration file at %s", configFile)
 	conf, err := config.Load(configFile)
 	if err != nil {
-		log.Printf("✗ Could not load configuration: `%v`", err)
-	}
-
-	// Log the loaded config
-	confBytes, err := json.MarshalIndent(conf, "  ", "  ")
-	if err != nil {
-		log.Printf("✗ Could not log config: `%v`", err)
+		log.Printf("✗ Could not load configuration file: `%v`", err)
 	} else {
-		if *showConfig {
-			log.Println("\n  " + string(confBytes))
+		// Log the loaded config
+		confBytes, err := json.MarshalIndent(conf, "  ", "  ")
+		if err != nil {
+			log.Printf("✗ Could not log config: `%v`", err)
+		} else {
+			if *showConfig {
+				log.Println("\n  " + string(confBytes))
+			}
+			log.Println("✓ Successfully loaded configuration")
 		}
-		log.Println("✓ Successfully loaded configuration")
 	}
 
 	var engine *series.Engine
@@ -79,6 +79,8 @@ func main() {
 			log.Fatal(err)
 		}
 		log.Println("✓ Successfully started series engine")
+	} else {
+		log.Println("  Not using series engine because -series-data-dir is not defined")
 	}
 
 	globalMessages := message.NewMessageChannel()
@@ -93,6 +95,9 @@ func main() {
 	for m := range globalMessages {
 		switch m.Class {
 		case series.SeriesEngineClassName:
+			if engine == nil {
+				continue
+			}
 			engine.Process(m)
 		default:
 			// Drop

